src/main/java/uk/gov/hmcts/ccd/util/ClientContextUtil.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.ccd.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class ClientContextUtil {
+    public static Logger LOG = LoggerFactory.getLogger(ClientContextUtil.class);
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    public static String mergeClientContexts(String jsonA, String jsonB) {
```

Comment:
My examples of `A`/`B` probably shouldn't be in the production code, as it's not every clear from the method signature which way round things are. First thing that comes to mind, but feel free to pick something more suitable:
```suggestion
    public static String mergeClientContexts(String originalJson, String mergeWithJson) {
```